single-node/user-data

```diff
@@ -83,7 +83,7 @@ function init_templates {
         mkdir -p $(dirname $TEMPLATE)
         cat << EOF > $TEMPLATE
 [Service]
-ExecStartPre=/usr/bin/mkdir -p /etc/kubernetes/manifests
```

Comment:
I moved the static manifests to `/etc/kubernetes/addons`, and the kube-system stuff is in `/srv/kubernetes/addons`. I'll defer to you on correctness of that decision.